__tests__/app.test.js

```diff
@@ -171,15 +171,31 @@ describe('GET /api/articles', () => {
         articles.forEach((article) => {
           expect(article).toEqual(
             expect.objectContaining({
+              article_id: expect.any(Number),
               author: expect.any(String),
               body: expect.any(String),
```

Comment:
check the trello for what we expect in this endpoint, don't want to see the body when getting all articles.